toolsWebsite/tools/templates/tools/index.html

```diff
@@ -110,6 +110,8 @@ <h4>{{ category.type }}</h4>
               Checked Out: {{ category.unavailable }}
               <br>
               Rental fee: ${{ category.price }}
+              <br>
+              <a href="addToCart/{{category.id}}">Add to Cart</a>
```

Comment:
i think we should add an `onClick` method here that doesn't allow a user to add a tool to the cart if there are 0 tools available

Reply:
I can do an onclick or just put that logic in the view.  Which would we prefer?

Reply:
The onclick is nice because it only navigates to the link if the criteria is met, so we wouldn't need to connect to the server unless it's okay. That's my thought anyways. You can look and see how I did it in the toolpage.html

Reply:
I didn't do an onclick, but I did do use the django template language to only render the checkout icon if there are tools available.  I'll make a new PR in a bit and you can check it out and see if it does what we are expecting!